src/Chunk.h

```diff
@@ -152,15 +152,21 @@ class cChunk :
 
 	cWorld * GetWorld(void) const { return m_World; }
 
-	void SetBlock(int a_RelX, int a_RelY, int a_RelZ, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, bool a_SendToClients = true);
+	void SetBlock(Vector3i a_RelBlockPos, BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, bool a_SendToClients);
```

Comment:
Why is the default being removed?

Reply:
I was trying to see how many call sites actually used the parameter set to `false`, and it slipped through. I guess it can stay now, it makes the callers think "do I need to send this change to the clients or not?"

Reply:
I feel like it's very rare to ever pass `false`. Not sure I could think of an example.

Reply:
Exactly, I'd say the param could be removed. But don't want to go there in this PR.

Reply:
If the end goal is to remove the parameter, why add it back in to all of the call sights?

Reply:
Alright, so I went ahead and checked, there's not a single place in Cuberite that would use `false`; however, the function is exported to Lua through `cWorld:SetBlock`, and I can't vouch for the plugins. However, since it makes no sense to not notify clients, I'm removing the parameter altogether.

Reply:
Should we keep the parameter in the lua version of `cWorld:SetBlock` and just ignore the value?

Reply:
And mark as OBSOLETE of course.

Reply:
No, let it burn.